qemu/tests/cfg/hugepage_specify_node.cfg

```diff
@@ -6,3 +6,56 @@
     not_preprocess = yes
     mem = 4096
     idle_node_mem = 1024
+    variants:
+        - 64k:
+            only aarch64
+            kernel_hp_file = "/sys/devices/system/node/node0/hugepages/hugepages-64kB/nr_hugepages"
```

Comment:
IMO, it is not make sense to use the file that include the node info(`node0`)  as a  `kernel_hp_file`.

In avocado, `kernel_hp_file` is also used to get the available_hugepages - `target_hugepages`. It is for all nodes.

Reply:
has been corrected